release notes/v0.38.0.md

```diff
@@ -0,0 +1,166 @@
+  ],
+};
+```
+
+Thanks, @Gabrielopesantos, for the [contribution](https://github.com/grafana/k6/pull/2488).
+
+### Improve JSON output's performance ([#2436](https://github.com/grafana/k6/pull/2436))
+
+The JSON output was optimized and now should around 2x as more performant at outputting metrics.  This means that it either will be able to export twice as many metrics and to use half the resources to do the same amount of metrics. As a side effect, there is a slight breaking change in which the `tags` field is no longer sorted, while it was previously.
+
+### Treat panics as interrupt errors ([#2453](https://github.com/grafana/k6/pull/2453))
+
+We changed the behavior of how k6 treats Go panics, which may happen because of bugs in k6 or in a JavaScript k6 extension. Previously, the behavior was to catch the panic and log it as an error. Starting with **v0.38.0**, whenever k6 observes a Go panic, it logs an error like before, but more importantly, it will abort the script execution and cause k6 to exit with a non-0 exit code. This will help extension authors to identify issues in their extensions more easily. 
+
+### Miscellaneous
+
+* [#2411](https://github.com/grafana/k6/pull/2411) The k6 command-line UI (logo, test description and progerss bars) can now effectively be disabled using the [`--quiet`](https://k6.io/docs/using-k6/options/#quiet) flag.
+* [#2429](https://github.com/grafana/k6/pull/2429) `lib/types` now exposes the source of the `NullHostnameTrie` to simplify access to an original list of the hostnames.
+
+* [#2484](https://github.com/grafana/k6/pull/2484) moved out in a new dedicated Go `lib/netext/grpcext` package all the parts not strictly required from the `js/k6/net/grpc` module for binding the gRPC operations and the JavaScript runtime. It facilitates the development of extensions based on gRPC without the direct dependency on the `goja` runtime. Furthermore, the new [Dial](https://github.com/grafana/k6/blob/bef458906f6884a99843573223028981c0a8b8db/lib/netext/grpcext/conn.go#L72-L81) function accepts a [grpc.DialOption](https://pkg.go.dev/google.golang.org/grpc@v1.45.0#DialOption) variadic for customizing the dialing operation.
+
+## Extensions
+
+### Event loop testing
+
+With this release, you can [export the event loop](https://pkg.go.dev/go.k6.io/k6/js/eventloop) added in v0.37.0. This lets extensions developers test event-loop-dependant APIs.
+
+There were also updates to [modulestest.VU](https://pkg.go.dev/go.k6.io/k6/js/modulestest#VU) to support the new API. Head to GitHub to [see it in action](https://github.com/grafana/k6/tree/master/cmd/integration_tests/testmodules/events).
+
+
+## Bugs Fixed!
+
+* [#2456](https://github.com/grafana/k6/pull/2456): Fixed a very unlikely panic involving arrival rate executors and execution segments with very low VU counts.
+* [#2349](https://github.com/grafana/k6/issues/2349): Thanks to @rainingmaster it is now possible to leave the options' tlsAuth `domains` property empty.
```

Comment:
What do you think about providing the links to the contributor's GitHub (here and in other places)? Like we did in previous release notes?

Reply:
Last time I wrote release notes @codebien told me it would be done automagically by GitHub. I believe it worked. Let me know if that's not the case.

Reply:
Double-checked and you're (and @codebien) absolutely right here :+1: 

It doesn't preview link in a View file mode, but which is more important, it produces a special section `Contributors` in the release notes. And for example for the [last release](https://github.com/grafana/k6/releases/tag/v0.37.0) there is no section since I tried to make the links :cry: